precompiles/staking/utils_test.go

```diff
@@ -465,7 +325,7 @@ func (s *PrecompileTestSuite) assertRedelegationsOutput(data []byte, redelTotalC
 // the same amount is considered for all entries
 func (s *PrecompileTestSuite) assertRedelegation(res staking.RedelegationResponse, entriesCount int, expValSrcAddr, expValDstAddr string, expAmt *big.Int, expCreationHeight int64) {
 	// check response
-	s.Require().Equal(res.Redelegation.DelegatorAddress, sdk.AccAddress(s.address.Bytes()).String())
+	s.Require().Equal(res.Redelegation.DelegatorAddress, sdk.AccAddress(s.keyring.GetAddr(0).Bytes()).String())
```

Comment:
unnecessary conversion


```suggestion
	s.Require().Equal(res.Redelegation.DelegatorAddress, s.keyring.GetAccAddr(0).String())
```